vaadin-dialog-flow-parent/vaadin-dialog-flow-integration-tests/src/test/java/com/vaadin/flow/component/dialog/tests/DialogTestPageIT.java

```diff
@@ -339,25 +339,41 @@ public void verifyResizeLimitsAreRespected() {
 
     @Test
     public void resizableDialogListenerIsCalled() {
+        findElement(
+                By.id("dialog-resizable-draggable-set-initial-position-button"))
+                .click();
         findElement(By.id("dialog-resizable-draggable-open-button")).click();
         WebElement message = findElement(
                 By.id("dialog-resizable-draggable-message"));
 
-        Assert.assertEquals("Initial size with width (200) and height (200)",
+        Assert.assertEquals(
+                "Initial size with top (50), left (50), width (200) and height (200)",
                 message.getText());
 
         TestBenchElement overlayContent = getOverlayContent();
 
         resizeDialog(overlayContent, 50, 50);
 
         Assert.assertEquals(
-                "Resize listener called with width (250) and height (250)",
+                "Resize listener called with top (50), left (50), width (250) and height (250)",
+                message.getText());
+
+        resizeDialog(overlayContent, -50, -50, "nw");
+
+        Assert.assertEquals(
+                "Resize listener called with top (0), left (0), width (300) and height (300)",
                 message.getText());
     }
 
     private void resizeDialog(TestBenchElement overlayContent, int xOffset,
             int yOffset) {
-        WebElement resizerSE = overlayContent.$(".resizer.se").first();
+        resizeDialog(overlayContent, xOffset, yOffset, "se");
+    }
+
+    private void resizeDialog(TestBenchElement overlayContent, int xOffset,
+            int yOffset, String direction) {
+        WebElement resizerSE = overlayContent.$(".resizer." + direction)
```

Comment:
`SE` was probably meant to indicate the south/east resizer. Now that it's configurable it can be confusing what that means. Maybe just name the variable `resizer`.

Reply:
Name changed.

Reply:
@DiegoCardoso To clarify, I was suggesting to rename `resizerSE` into `resizer`.

Reply:
Ah, my bad 😅
Changed it to `resizer`